src/components/Notifiction/Notification.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+import { Error } from '../../types/Error';
+import { useTodo } from '../../api/useTodo';
+
+export const Notification: React.FC = () => {
+  const { errorMessage, setErrorMessage } = useTodo();
+  let timeoutId: number;
```

Comment:
Save timeoutId in useRef
```suggestion
  const timeoutId = useRef<number>(0);
```